src/infer_request.h

```diff
@@ -712,6 +713,16 @@ class InferenceRequest {
     return Status::Success;
   }
 
+  bool IsCancelled()
+  {
+    bool is_cancelled = false;
+    Status status = IsCancelled(&is_cancelled);
+    if (!status.IsOk()) {
+      LOG_ERROR << status.Message();
+    }
+    return is_cancelled;
+  }
+
 #endif  // TRITON_ENABLE_STATS
```

Comment:
I did not notice this before - but looks like the Cancel and isCancel methods are within the TRITON_ENABLE_STATS ifdef - and should be moved out.

Reply:
Thanks for catching this. It is moved out.